autogen/agentchat/groupchat.py

```diff
@@ -248,6 +251,15 @@ def __post_init__(self):
         elif self.max_retries_for_selecting_speaker < 0:
             raise ValueError("max_retries_for_selecting_speaker must be greater than or equal to zero")
 
+        # Load message transforms here (load once for the Group Chat so we don't have to re-initiate it and it maintains the cache across subsequent select speaker calls)
+        if self.select_speaker_auto_message_transforms is not None:
+            if isinstance(self.select_speaker_auto_message_transforms, transform_messages.TransformMessages):
```

Comment:
I assume type hints should handle this check, not sure if it's needed

Reply:
Thanks for the note @WaelKarkoub, as Type hints aren't enforced I think it's best to keep this in place?